presto-jdbc/src/main/java/com/facebook/presto/jdbc/ConnectionProperties.java

```diff
@@ -169,6 +171,15 @@ public ApplicationNamePrefix()
         }
     }
 
+    private static class ClientTags
+            extends AbstractConnectionProperty<String>
+    {
+        public ClientTags()
+        {
+            super("clientTags", NOT_REQUIRED, ALLOWED, STRING_CONVERTER);
```

Comment:
Not sure fully but maybe construct a new converter? @tdcmeehan I know we had the discussion of not creating a converter unless necessary but STRING_CONVERTER just takes it as it is without validation so it could cause problem if someone passes a wrong string, wdyt?

Reply:
Agree, a converter that does client-side validation would be great.

Reply:
@tdcmeehan @basarhamdionat what  are the exact expectations from this new converter?
When can we expect this to get merge?
If possible, can we expedite this?

Reply:
@Ayan07 reading @basarhamdionat's message, it seems like it would be a good idea to add a new converter (examples in this class) that validates the input is a comma-separated alphanumeric series of tags, and fails with a descriptive error message if not.

Happy to approve and merge once all the feedback is addressed.

Reply:
@tdcmeehan okay, got the context.
Will work on it and  raise a new PR.
Also, do we have any reference(in any other class) where such a validation has been implemented?